### ISSUE TITLE
Plug-in options will not load

### ISSUE THREAD

**Issue by TemporaryChan** [SeriousCache/UABE#330]
Hi, there. I'm experiencing a bug that appeared too suddenly. Recently, in attempt to changing textures in a game's files, the UABE is having problems loading the options I use to export and import the textures in the plug-ins. Whenever I click the plug-in option, it just enters a blank state that allows the export options to not be given. I tried to redownload or use a different version of the UABE. However, the problem still remained. As far as I can see, it seems as though I'm the only one experiencing this problem, so I need to know if there is something I did that may have caused this in the first place. 
![IMG_20200308_170214](https://user-images.githubusercontent.com/61955846/78729447-11968d80-7908-11ea-87ac-0d72f2cc1520.jpg)

**Comment by TemporaryChan** [comment]
This is still an issue. I first experienced this the moment I extracted an audio sound. Ever since then, it didn't matter how many versions of this Bundle Extractor I download, the same problem happens over and over. I've been experiencing this for months, and I'm getting fed up. I cannot finish my mod if I'm going to keep experiencing this problem. I even tried looking to see if anyone else experienced this so I could find a solution, but it seems as though this is only my problem. Please, help me. I have absolutely no other way to import textures back into my game.

**Comment by DerPopo** [comment]
The plugins will only show up if all selected assets are supported. The texture plugin will only show up if all assets are detected as Texture2D. Materials are not supported by any of the standard plugins. 

Does UABE ask about selecting a type database when opening the file? Since there is no type databse for Unity 2019.3 so far, some issues may arise with the new files.

Another cause may be anti virus tools preventing UABE from loading the plugins. While it seems like the standard answer to any sort of problem, I've seen these tools do weirder things than that without telling the user.

**Comment by TemporaryChan** [comment]
The file I'm trying to open is a texture 2d file, but maybe it is the anti virus tool I have installed. As far as the Unity 2019 situation, it didn't ask me about selecting a type database. When I first used it, it allowed me to export and import textures for like it was supposed to but only for a short period of time before the problem happened in a 2019 build of the game. But when I tested it out on an old 2017 build of my game, it works completely normal.
